main.mjs

```diff
@@ -17,6 +17,10 @@ function exit(message) {
 }
 
 async function main() {
+  core.warning(
+    "Running on setup-deno@1, which is deprecated.\nPlease update to setup-deno@2, which defaults to Deno 2.0",
```

Comment:
```suggestion
    "Running on setup-deno@v1, which is deprecated.\nPlease update to setup-deno@v2, which defaults to Deno 2.0",
```

Add a link to more info. That link should mention that you can use 1.x with setup-deno@v2`.